tests/suites/0_stateless/20+_others/20_0012_privilege_access.sh

```diff
@@ -172,6 +172,16 @@ echo "select count(1) from information_schema.columns where table_schema in ('in
 echo "select count(1) from information_schema.tables where table_schema in ('information_schema', 'system');;" | $USER_A_CONNECT
 echo "select count(1) from information_schema.tables where table_schema not in ('information_schema', 'system');" | $USER_A_CONNECT
 
+echo "=== test UDF priv"
+echo "DROP FUNCTION IF EXISTS test_alter_udf;" |  $MYSQL_CLIENT_CONNECT
```

Comment:
Can we disable `$MYSQL_CLIENT_CONNECT`?

Reply:
Yes , but this pr also in draft. I will modify it before ready for review.

Reply:
Another issue is that we should spilt the privilege related tests into many files by the module, like:
```
/0_stateless/18_rbac/udf_priv.sh
/0_stateless/18_rbac/udf_priv.result
```